src/hitbox.js

```diff
@@ -61,7 +57,7 @@ var HitBox = function() {
 	};
 };
 
-helpers.extend(HitBox.prototype, {
+Object.assign(HitBox.prototype, {
```

Comment:
This will not work in IE11. Could probably use `Object.setPrototypeOf(HitBox.prototype, {` instead.